test/functional/page_objects/home_page.ts

```diff
@@ -58,8 +58,8 @@ export class HomePageObject extends FtrService {
 
   async isWelcomeInterstitialDisplayed() {
     // give the interstitial enough time to fade in
-    await new Promise((resolve) => setTimeout(resolve, 500));
-    return await this.testSubjects.isDisplayed('homeWelcomeInterstitial', 2000);
+    await new Promise((resolve) => setTimeout(resolve, 1000));
+    return await this.testSubjects.isDisplayed('homeWelcomeInterstitial', 5000);
```

Comment:
Not against the change, but if it is a page loading issue, you might need to update test with wait for global loading indicator to disappear:

```
      await PageObjects.common.navigateToUrl('home', undefined, { disableWelcomePrompt: false });
      await PageObjects.header.waitUntilLoadingHasFinished();
      expect(await PageObjects.home.isWelcomeInterstitialDisplayed()).to.be(true);
```

Sometimes it may take a while to finish loading the page.

Reply:
Thanks for taking a look @dmlemeshko - I've reverted the timeout bump and pulled in the header loading check instead. Rerunning flaky test runner here: https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/6652